tests/Feature/CodeGenerator/MakeFileCommandTest.php

```diff
@@ -4,33 +4,39 @@
 
 namespace GacelaTest\Feature\CodeGenerator;
 
+use Gacela\Console\Infrastructure\Command\MakeFileCommand;
+use Gacela\Framework\Gacela;
 use GacelaTest\Feature\Util\DirectoryUtil;
 use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\StringInput;
+use Symfony\Component\Console\Output\BufferedOutput;
 
 final class MakeFileCommandTest extends TestCase
 {
-    private const ENTRY_POINT = __DIR__ . '/../../../';
-
     public static function tearDownAfterClass(): void
     {
-        DirectoryUtil::removeDir(self::ENTRY_POINT . 'src/TestModule');
+        DirectoryUtil::removeDir('./src/TestModule');
     }
 
     public function setUp(): void
     {
-        DirectoryUtil::removeDir(self::ENTRY_POINT . 'src/TestModule');
+        Gacela::bootstrap(__DIR__);
+        DirectoryUtil::removeDir('./src/TestModule');
     }
 
     /**
      * @dataProvider createFilesProvider
      */
     public function test_make_file(string $action, string $fileName, string $shortName): void
     {
-        $command = sprintf('%sgacela make:file %s Gacela/TestModule %s', self::ENTRY_POINT, $shortName, $action);
-        exec($command, $output);
+        $input = new StringInput(sprintf('%s Psr4CodeGenerator/TestModule %s', $shortName, $action));
+        $output = new BufferedOutput();
+
+        $command = new MakeFileCommand();
+        $command->run($input, $output);
```

Comment:
This is the important part. Instead of using `exec(...)` you need to instantiate the command and run it properly, so PHPUnit is aware about the actual code coverage 🧠